app-crypt/steghide/steghide-0.5.1.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A Steganography Program"
+DESCRIPTION="Steghide is a steganography program that \
+is able to hide data in various kinds of image and \
+audio files. The color respectivly sample-frequencies \
+are not changed thus making the embedding resistant \
+against first-order statistical tests."
```

Comment:
You can put up to 80 characters per line. Here's what we can use:
```
DESCRIPTION="Steghide is a steganography program that is able to hide data in \
various kinds of image and audio files. The color respectivly \
sample-frequencies are not changed thus making the embedding resistant against \
first-order statistical tests."
```

@alaviss, thanks for your review! That's cool :)

BTW, I'm getting a different sha256sum for the tar.bz2:
```
CHECKSUM_SHA256="a2c7f879a3e22860879889106cc49e486000653f81448264affa0fd616a47da1"
```

Reply:
Yes, can confirm it's `a2c7f879a3e22860879889106cc49e486000653f81448264affa0fd616a47da1`. Should've double-check the checksum first.